project-base/app/config/public.key

```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
+MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA4Y3boig3D3Si/OxTJfGz
+2bm68lTydupL0f5vFU1pPfQ73eKRp7oFH2i5TNKtUY5Ys48Sr/iyv4s10YhjhouJ
+mt/3OvaFD09yo07Vi7osoPqFZWIc84Z0lNmZxRjNhD5K3eIqKGBE9894mpSG4Ooi
+Wv610RvVdHOLNLq/tQtBWHwILPS/vkSn5pZxcBaLX285s5qGXskBLrbuU9oqZvfI
+ArXxNhrVuGlhmMFKJ0cFOJ45OTtpogBQYaLt0m8AN4RDE9Qy11t88mR2pLfHJlyx
+3gw2kh5itL9GkKsnfNS+zxh+2XqOG9oLsvtNskQEyjEaBXUX3r7B1lulHTibvnyl
+1QIDAQAB
+-----END PUBLIC KEY-----
```

Comment:
I suggest a command to generate a key pair.

Reply:
@TomasLudvik ;-)